testing/conformance/tests/runner.rs

```diff
@@ -276,10 +303,33 @@ async fn run_vector(
 ) -> anyhow::Result<impl Iterator<Item = impl Future<Output = anyhow::Result<VariantResult>>>> {
     let file = File::open(&path)?;
     let reader = BufReader::new(file);
-    let vector: TestVector = serde_json::from_reader(reader)?;
 
-    match vector {
-        TestVector::Message(v) => {
+    // Test vectors have the form:
+    //
+    //     { "class": ..., rest... }
+    //
+    // Unfortunately:
+    // 1. That means we need to use serde's "flatten" and/or "tag" feature to decode them.
+    // 2. Serde's JSON library doesn't support arbitrary precision numbers when doing this.
```

Comment:
That's unfortunate.

Reply:
Is there an issue for this upstream?

Reply:
I believe https://github.com/serde-rs/serde/issues/1183 is best, but there are a bunch of related issues. But I'll link to that one.